package.json

```diff
@@ -2,14 +2,14 @@
   "name": "@openmrs/style-referenceapplication",
   "version": "0.1.0",
   "description": "Styling (css, fonts, images) for Reference Application",
-  "main": "lib/referenceapplication.js",
+  "main": "index.js",
```

Comment:
It shouldn't be necessary since we are pointing main to lib/referenceapplication.js, which is created from https://github.com/openmrs/openmrs-web-style-referenceapplication/blob/master/src/index.js

Did you have trouble making it work otherwise?

Reply:
No, it worked fine. I was thinking if the file would exist, since the lib folder is auto generated. 
But now you mentioned it, i will revert back

Reply:
Cool.